### ISSUE TITLE
Allow sorting missions by total reward

### ISSUE THREAD

**Issue by sirzooro** [TLDRMissions/TLDRMissions#524]
**TLDR Version of your idea**
When alt does not have Torghast companions unlocked or many of them are low health and cannot be sent, I I would like to send missions with more/better rewards first.

**Slightly longer version**
Please add new sorting criteria: more/better rewards first. I would like it as a 3rd option in existing option "Hardest or Easiest mission first". Also it would be nice as a 3rd sorting criteria what this option is set to Hardest or Easiest and there are 2 or more missions with equal difficulty - in such cases TLDR could start from one with best rewards.

I propose following criteria to determine which mission is better:
- Gold - more gold;
- Anima - more anima (compare total anima, not number of items);
- Follower XP items - better items first;
- Pet Charms - more charms first;
- Augment Runes - more runes first;
- Reputation Tokens - more reputations first (when mission offers multiple reps, sum them first);
- Bonus Follower XP - more XP first;
- Crafting Materials - Zereth Mortis bags first, Korthia second, them everything else. In every of these group Rare should be first, then Uncommon and Common as last one;
- Runecarver Crafting Currencies - more currency first;
- Campaign Mission Progress - none;
- Equipment - ones with highest vendor price first;
- Sanctum Feature - sort by rarity, then by item count.

**Comment by teelolws** [comment]
Hardest missions *are* missions with better rewards. It sorts on the missions level, and if their level ties, it sorts on how much the reward is.
